File: android/src/main/kotlin/com/imf/blade/container/nativeEvents/NativeEventListener.kt
package com.imf.blade.container.nativeEvents

interface NativeEventListener {
    fun pushFlutterPage(event: PushFlutterPageEvent)
    fun pushNativePage(event: PushNativePageEvent)
    fun popNativePage(event: PopNativePageEvent)
    fun popUntilNativePage(event: PopUntilNativePageEvent)
}

File: ios/Classes/protocol/NativeEventListener.swift
//
//  NativeEventListener.swift
//  blade
//
//  Created by sangya on 2021/8/31.
//

import Foundation
public protocol NativeEventListener {
    func pushFlutterPage(event: FlutterBaseEvent)
    func pushNativePage(event: FlutterBaseEvent)
}
